metrics/metrics_test.go

```diff
@@ -0,0 +1,52 @@
+package metrics
+
+import (
+	"testing"
+
+	"github.com/prometheus/client_golang/prometheus/testutil"
+	"github.com/stretchr/testify/assert"
+	"sigs.k8s.io/controller-runtime/pkg/metrics"
+)
+
+var defaultRegistry = metrics.Registry
+var find string = "odf_system_map"
+
+func TestReportODFSystemMapMetrics(t *testing.T) {
+	type args struct {
+		storageSystem, name, namespace, kind string
+	}
+	tests := []struct {
+		name string
```

Comment:
can you pls make this `name` as a `label`. just want to follow the same pattern everywhere.